pycobertura/reporters.py

```diff
@@ -95,6 +95,9 @@ def generate(self):
 
 class HtmlReporter(TextReporter):
     def __init__(self, *args, **kwargs):
+        self.title = kwargs.pop("title", "pycobertura report")
+        self.inline_sources = kwargs.pop("inline_sources", True)
+        self.sources_message = kwargs.pop('sources_message', None)
```

Comment:
Can we have a fallback message in case no message is provided to prevent confusion? Also, can we also rename `no_file_sources_message`.

```suggestion
        self.no_file_sources_message = kwargs.pop(
            "no_file_sources_message", "Rendering of source files was disabled."
        )
```

But for your use-case, do you really need to customize the message? Wouldn't a generic message "Rendering of source files was disabled" satisfy your needs? Why do you need to customize it in the first place?